### PR TITLE
Oglle Presence 0.0.1

### PR DESCRIPTION
**Pull request by MateusOFCZ**

### CODE REVIEW

**Comment by Bas950** [comment]
Why did you open a new PR...

File: Oglle/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+		"name": "Leprechaun",
+		"id": "394147438006370304"
+	},
+	"service": "OGLLE.COM.BR",
+	"description": {
+        "pt_BR": "Precisa de um BOT para Discord? Edição/criação de sites? Ou qualquer outro serviço fora do comum? Este é o site feito para você!",
+        "en": "Need a BOT for Discord? Website editing/creation? Or any other out-of-the-ordinary service? This is the site made for you!"
+	},
+	"url": ["oglle.com.br"],
+	"version": "0.0.1",
```
Comment (Bas950) [inline Oglle/dist/metadata.json]: url doesnt need to be in an array
Version 1.0 not 0.0.1

File: Oglle/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
+{
+	"author": {
+		"name": "Leprechaun",
+		"id": "394147438006370304"
+	},
+	"service": "OGLLE.COM.BR",
```
Comment (Bas950) [inline Oglle/dist/metadata.json]: Service should just be "Oggle"

File: Oglle/dist/presence.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+        presenceData.state = "Serviços"
+    }
+    if(window.location.pathname.includes("/noticias")) {
+        presenceData.state = "Notícias"
+    }
+    if(window.location.pathname.includes("/#Contato")) {
+        presenceData.state = "Contato"
+    }
+    if(window.location.pathname.includes("/anuncios")) {
+        presenceData.state = "Ganhando dinheiro com a Oglle.com.br!"
```
Comment (Bas950) [inline Oglle/dist/presence.js]: it should be .details not .state if you only set 1

**Comment by MateusOFCZ** [comment]
> Why did you open a new PR...
I can't reopen the previous one, as the repository has been deleted!

**Print:**
image

File: Oglle/dist/metadata.json [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+	"author": {
+		"name": "Leprechaun",
+		"id": "394147438006370304"
+	},
+	"service": "Oglle",
+	"description": {
+        "pt_BR": "Precisa de um BOT para Discord? Edição/criação de sites? Ou qualquer outro serviço fora do comum? Este é o site feito para você!",
+        "en": "Need a BOT for Discord? Website editing/creation? Or any other out-of-the-ordinary service? This is the site made for you!"
+	},
+	"url": ["oglle.com.br"],
```
Comment (Bas950) [inline Oglle/dist/metadata.json]: url doesnt need to be in an array

**Pull request merged**